react-pr2/src/store/Slice/cartSlice.js

```diff
@@ -0,0 +1,43 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const cartSlice = createSlice({
+  name: 'carts',
+  initialState: {
+    showMiniCart: false,
+    cartItem: [],
+  },
+  reducers: {
+    setShowMiniCart(state) {
+      state.showMiniCart = true;
+    },
+    setHideMiniCart(state) {
+      state.showMiniCart = false;
+    },
```

Comment:
```suggestion
    toggleMiniCart(state, action) {
      state.showMiniCart = action.payload;
    },
```
`toggleMiniCart(true)` hoặc `toggleMiniCart(false)`